react/LocationCountry.tsx

```diff
@@ -51,6 +51,7 @@ const renderCountryFlagWithName = ({
 )
 
 const LocationCountry: React.FC = () => {
+  const labelId = 'location-country'
```

Comment:
this could be a prop if you'd ever want to change this label

Reply:
sure